server/tests/index.js

```diff
@@ -690,3 +690,67 @@ describe('create a request', () => {
   });
 });
 
+// Test to modify a user's Request
+describe('UPDATE a user request', () => {
+  it('it should GET update a users request', (done) => {
+    request(app)
+      .put('/api/v1/users/requests/2')
+      .set('x-access-token', token)
+      .send({
+        requestTitle: 'fix the tv now',
+        requestBody: 'the request body is required noejur',
+      })
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body.data).to.haveOwnProperty('request');
```

Comment:
In your test, test everything you are expecting to receive. Hence, I expect to see you test the content of the body being returned